scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content"
+
+# If the docs directory is not present, generate docs
+if [ ! -d $docsPath ]; then
+  $(npm bin)/gulp docs
+fi
+
+# Get git meta info for commit
+commitSha="$(git rev-parse --short HEAD)"
+commitAuthorName="$(git --no-pager show -s --format='%an' HEAD)"
+commitAuthorEmail="$(git --no-pager show -s --format='%ae' HEAD)"
+commitMessage="$(git log --oneline -n 1)"
+
+# create directory and clone test repo
+rm -rf $tempRepoPath
+mkdir $tempRepoPath
+git clone $repoUrl $tempRepoPath
+
+# Clean out repo directory and copy contents of dist/docs into it
+rm -rf $tempRepoPath/*
+mkdir $tempRepoPath/docs
```

Comment:
Realized I should add a bit more here; we should:
* Change the current `gulp docs` task to be called `gulp markdown-docs`
* Change the current `gulp api` to `gulp api-docs`
* Add a new task called `gulp docs` that just runs the previous two
* Change `gulp api-docs` to output to `dist/docs/api` and `markdown-docs` to output to `dist/docs/markdown`

This should make it easier for this script to massage the different types of docs into the right place (and still only need to run one command).